tests/cli.rs

```diff
@@ -191,6 +192,7 @@ fn compare_files(expect_path: impl AsRef<Path>, got_dir_path: impl AsRef<Path>)
 ///
 /// Errors on the first regex mis-match or if the stderr output has fewer lines than expected
 fn match_get_stderr(stderr: Vec<u8>) -> Result<()> {
+    println!("{}", String::from_utf8_lossy(&stderr[..]));
```

Comment:
Regardless of which writer fix we take, the changes in this file are still useful.